base/system/diskpart/diskpart_msg.mc

```diff
@@ -7219,84 +7237,87 @@ Example:
     SET ID=ebd0a0a2-b9e5-4433-87c0-68b6b72699c7
 .
 Language=Romanian
-    Changes the partition type field for the partition with focus.
+    Schimbă registru tipului partiției pentru partiția în cauză.
 
-Syntax:  SET ID={<BYTE> | <GUID>} [OVERRIDE] [NOERR]
+Sintaxă:  SET ID={<BYTE> | <GUID>} [OVERRIDE] [NOERR]
 
     ID={<BYTE> | <GUID>}
 
-                Specifies the new partition type.
-                For master boot record (MBR) disks, you can specify a partition
-                type byte, in hexadecimal form, for the partition. Any
-                partition type byte can be specified with this parameter except
-                for type 0x42 (LDM partition). Note that the leading '0x' is 
-                omitted when specifying the hexadecimal partition type.
-
-                For GUID partition table (GPT) disks you can specify a
-                partition type GUID for the partition. Recognized GUIDs
-                include:
-
-                    EFI System partition:
+                Specifică noul tip al partiției.
+                Pentru discurile cu Registrul principal de inițializare (MBR),
+                puteți speciifica o partiție de tip octet, în formă
+                hexazecimală, pentru partiție. Orice partiție de tip octet
+                poate fi specificată cu acest parametru, exceptând pentru tipul
+                0x42 (partiție LDM). Rețineți că primul '0x' este omis când
+                se specifică partiții de tip hexazecimal.
+                                                                
+                Pentru discurile cu tabelul de partiție GUID (GPT) puteți
+                specifica un tip de partiție GUID pentru partiția pe care vreți
+                să o creați. GUID-urile recunoscute includ:
+
+                    Partiția de sistem EFI:
                         c12a7328-f81f-11d2-ba4b-00a0c93ec93b
 
-                    Basic data partition:
+                    Partiția de date de bază:
```

Comment:
I'm quite sure that [basic data partition](https://en.wikipedia.org/wiki/Microsoft_basic_data_partition) does not have "basic" describing the "data", as I read it in Romanian translation. You already found [a better formula](https://github.com/reactos/reactos/pull/5485/files#diff-630911a34e337d56b06d0a775033350a10d580f71e0cfe87b3b6104a429bedfbR2109).

Reply:
The [referenced formula](https://github.com/reactos/reactos/pull/5485/files#diff-630911a34e337d56b06d0a775033350a10d580f71e0cfe87b3b6104a429bedfbR2108) was "partiție de bază pentru date", not "partiția de bază de date"!